src/ctap/storage.rs

```diff
@@ -374,11 +366,10 @@ impl PersistentStore {
                 _deserialize_min_pin_length_rp_ids(&value)
             });
         debug_assert!(rp_ids.is_some());
-        Ok(rp_ids.unwrap_or(vec![]))
+        Ok(rp_ids.unwrap_or_default())
     }
 
     /// Sets the list of RP IDs that are used to check if reading the minimum PIN length is allowed.
-    #[cfg(feature = "with_ctap2_1")]
     pub fn _set_min_pin_length_rp_ids(
```

Comment:
why are these (this function and the next 2 function show in this diff) unused? Is it WAI?

Reply:
They are used in #246. Basically, implementing new commands is pretty hard to split into reasonably sized PRs.